app/helpers/security_helper.rb

```diff
@@ -36,6 +36,6 @@ def add_stream_url(stream_info)
   private
 
     def not_checked_out?(media_object_id)
-      Avalon::Configuration.controlled_digital_lending_enabled? && Checkout.checked_out_to_user(media_object_id, current_user&.id).empty?
+      MediaObject.find(media_object_id).cdl_enabled && Checkout.checked_out_to_user(media_object_id, current_user&.id).empty?
```

Comment:
Seems like it is. Some of the rspec tests initially didn't like it but an `include CdlHelper` cleared it up. 

Specifically, it was the master_files_controller_spec that I saw having issues. It would cause an error about the MasterFileController not implementing `lending_enabled?`. The `include` line isn't strictly necessary if I'm understanding how views and helpers work, but I guess `render_views` in a controller spec doesn't quite treat or load things the same way as the application itself?